lib/index.js

```diff
@@ -172,6 +172,7 @@ Segment.prototype.normalize = function(msg) {
   msg.userId = msg.userId || user.id();
   msg.anonymousId = user.anonymousId();
   msg.sentAt = new Date();
+  extend(msg, this._extras);
```

Comment:
Can we rename this from `_extras` to something more descriptive? `_metadata` would work if it's available.